content/courses/onchain-development/intro-to-onchain.md

```diff
@@ -67,7 +81,8 @@ execute.
 
 Instruction handlers are how blockchain programs process the instructions from
 clients. Every exchange, lending protocol, escrow, oracle, etc. provides their
-functionality by instruction handlers.
+functionality by instruction handlers. They enable program to process complex
+financial operations on Solana.
```

Comment:
Not sure re: "They enable program to process complex financial operations on Solana." - people already know this based on the previous sentence.

Reply:
y; that might be a bit redundant